app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -32,6 +33,15 @@ def rhsm_organization_label
         self.organization.label
       end
 
+      def info_with_katello
+        info = info_without_katello
+        info['parameters']['kt_env'] = self.lifecycle_environment.try(:label) #deprecated
+        info['parameters']['kt_cv'] = self.content_view.try(:label) #deprecated
```

Comment:
Yeah, for this case I don't think that makes sense as it will be called whenever a host fetches its puppet yaml regardless of whether its used or not.  I will work on adding a katello.org page